pyformance/registry.py

```diff
@@ -136,6 +136,9 @@ def timer(self, key, tags=None):
         :param tags: tags attached to the timer (e.g. {'region': 'us-west-1'})
         :type tags: C{dict}
 
+        :param sample: a sample manager, default is ExpDecayingSample
```

Comment:
consider stating this default in the function definition itself and not only in the comment.

Reply:
These classes are mutable, so it wouldn't be a good idea to use them as defaults in the function definition

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments